### PR TITLE
List available services

### PR DESCRIPTION
**Pull request by azrael783**
Added the new services due to the fact that I was not aware that they are existing.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +22,17 @@ Custom component for Home Assistant to interact with smart devices via the Somfy
 
 This integration doesn't rely on a hardcoded list of devices anymore, but relies on the characteristics of every device. This means that more devices will be supported out of the box, based on their category, available states and commands. If your device is not supported or working correctly, have a look [here](#device-not-supported--working-correctly).
 
+## Tahoma services
+
+After installation this integration adds new services to Home-Assistant which can be used in automations. The new services are:
+
++ Somfy TaHoma: Set cover position with low speed (tahoma.set_cover_position_low_speed)
```
Comment (tetienne) [inline README.md:29]: Is available only if supported by the device.

File: README.md [hunk 2]
```diff
@@ -22,6 +22,17 @@ Custom component for Home Assistant to interact with smart devices via the Somfy
...
++ Somfy TaHoma: My position (cover) (tahoma.set_cover_my_position)
++ Somfy TaHoma: Execute command (tahoma.execute_command)
++ Somfy TaHoma: Get execution history (tahoma.get_execution_history)
+
+If you want to move your covers always with low speed create templates for each cover which calls the first service.
```
Comment (tetienne) [inline README.md:34]: Can you please link the documentation page of the cover template?

**Comment by tetienne** [comment]
That's indeed a good idea to add these services in the documentation 🙏

**Review by tetienne — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -22,6 +22,17 @@ Custom component for Home Assistant to interact with smart devices via the Somfy
 
 This integration doesn't rely on a hardcoded list of devices anymore, but relies on the characteristics of every device. This means that more devices will be supported out of the box, based on their category, available states and commands. If your device is not supported or working correctly, have a look [here](#device-not-supported--working-correctly).
 
+## Tahoma services
```
Comment (iMicknl) [inline README.md:25]:
```suggestion
## Tahoma services
```

File: README.md [hunk 4]
```diff
@@ -22,6 +22,17 @@ Custom component for Home Assistant to interact with smart devices via the Somfy
...
+
+After installation this integration adds new services to Home-Assistant which can be used in automations. The new services are:
```
Comment (iMicknl) [inline README.md:27]:
```suggestion
After installation this integration adds new services to Home Assistant which can be used in automations. The new services are:
```

File: README.md [hunk 5]
```diff
@@ -22,6 +22,17 @@ Custom component for Home Assistant to interact with smart devices via the Somfy
...
+
++ Somfy TaHoma: Set cover position with low speed (tahoma.set_cover_position_low_speed, only added if supported by the device)
++ Somfy TaHoma: My position (cover) (tahoma.set_cover_my_position)
++ Somfy TaHoma: Execute command (tahoma.execute_command)
++ Somfy TaHoma: Get execution history (tahoma.get_execution_history)
+
+If you want to move your covers always with low speed create [templates](https://www.home-assistant.io/integrations/cover.template/) for each cover which calls the first service.
```
Comment (iMicknl) [inline README.md:34]: Maybe we can add an example of a template cover in a future PR? It would help a lot of users. Add it to the advanced section.

**Review by iMicknl — COMMENTED** [review]
Good improvement! Just some minor comments.

**Comment by azrael783** [comment]
Yes, I can add an example. Will anyway create another PR due the link in the readme which points to the repo but I guess should point to the issues of the repo ;-)

**Review by iMicknl — APPROVED** [review]

**Comment by TheFes** [comment]
I also have a service: service: tahoma.set_light_my_position which is not mentioned yet (and for which I wonder wat it does).
I also do not know what kind of commands can be executed with tahoma.execute_command.

**Pull request merged**